examples/aditof-demo/CMakeLists.txt

```diff
@@ -16,6 +16,14 @@ else()
 	endif()
 endif()
 
+if( WIN32 OR UNIX )
```

Comment:
Rethink the order. Dragonboard and Raspberry PI are Unix as well.

Reply:
done